### PR TITLE
	add override paths to instrument

### PR DESCRIPTION
**Pull request by giannic**
Hello @nicks, 

Please review the following commits I made in branch 'giannic/medium2'.

c91812913d4bb6ab3dcee01eb0951f0c7e5bba87 (2016-12-13 23:21:06 -0800)
always instrument files inside mono/medium2

R=@nicks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -240,7 +240,7 @@ NYC.prototype.walkAllFiles = function (dir, visitor) {
 }
 
 NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
-  var instrument = this.exclude.shouldInstrument(filename, relFile)
+  var instrument = filename.indexOf('mono/medium2') >= 0 || this.exclude.shouldInstrument(filename, relFile)
```
Comment (nicks) [inline index.js]: could you make this injectable via the config object? i don't want to have to change the library every time we need to add a new directory. We have JS tests outside of medium2.

**Comment by giannic** [comment]
added `mediumIncludePathsOverride`, I didn't want to overload the already existing `include` config. That's also passed over to `test-exlude` lib and used there.

File: index.js [hunk 2]
```diff
@@ -241,6 +241,10 @@ NYC.prototype.walkAllFiles = function (dir, visitor) {
 
 NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
   var instrument = this.exclude.shouldInstrument(filename, relFile)
+  instrument = instrument || this.config.mediumIncludePathOverrides.some(function (path) {
```
Comment (nicks) [inline index.js]: can you add a null check on mediumIncludePathOverrides?

**Review by nicks — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (giannic) [reply index.js]: agh, sorry. added.

**Pull request merged**